### DISCUSSION TITLE
How to suppress this information output by giscus?

### DISCUSSION THREAD

**Discussion by Octobug** [giscus/giscus-component#1897] [Q&A] [answered]
Hey guys,

I wonder if it's possible to suppress output in the browser console. As you can see, there is no output in the first demo. But for any other component demos, there are always console output.

Have no idea why they behave in different ways.

- https://giscus-component.vercel.app/ (no console output)
- https://giscus-component.vercel.app/vue (output with host info)

<img width="831" alt="image" src="https://github.com/giscus/giscus-component/assets/8007022/a3a797b7-421d-46ef-b607-69aa8e6d8e95">

<img width="834" alt="image" src="https://github.com/giscus/giscus-component/assets/8007022/e83950bb-373d-4e63-8e9b-22cb75d82c14">

**Comment by laymonage** [comment, marked as answer]
Thanks, this should be fixed in v1.4.0 of the web component (and v2.4.0 for the other components).

**Reply by Octobug** [reply]
It works! Thank you!
